packages/generator/src/function-import/import.ts

```diff
@@ -26,21 +26,24 @@ export function importDeclarations(
   );
   return [
     ...externalImportDeclarations(functionImportParameters),
-    coreImportDeclaration([
-      ...corePropertyTypeImportNames(functionImportParameters),
-      ...returnTypes.map(returnType =>
-        responseTransformerFunctionName(returnType)
-      ),
-      ...(returnTypes.some(
-        returnType =>
-          returnType.returnTypeCategory ===
-          VdmFunctionImportReturnTypeCategory.EDM_TYPE
-      )
-        ? ['edmToTs']
-        : []),
-      'FunctionImportRequestBuilder',
-      'FunctionImportParameter'
-    ]),
+    coreImportDeclaration(
+      [
+        ...corePropertyTypeImportNames(functionImportParameters),
+        ...returnTypes.map(returnType =>
+          responseTransformerFunctionName(returnType)
+        ),
+        ...(returnTypes.some(
```

Comment:
Minor: could you perhaps these 10 lines in a little helper function like responseTransformerFunctionName so that the code looks a bit cleaner.

Reply:
Can you specify the lines you mean?

Reply:
I now just assumed 35 and ongoing

Reply:
```
...(returnTypes.some(
          returnType =>
            returnType.returnTypeCategory ===
            VdmFunctionImportReturnTypeCategory.EDM_TYPE
        )
          ? ['edmToTs']
          : [])
```